src/main/java/springData/dto/OfficeRequest.java

```diff
@@ -0,0 +1,64 @@
+package springData.dto;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.math.BigDecimal;
+
+public class OfficeRequest {
+    @Min(value = 104, message = "1")
+    private BigDecimal office;
+
+    @Size(min = 3, max = 10, message = "2")
+    private String city;
+
+    @Size(min = 3, max = 10, message = "3")
+    private String region;
+
+    @Valid
+    @NotNull(message = "Not null under valid")
```

Comment:
if you use everything code - use code there too.  One of codes of internal  OfficeDetails validations

Reply:
4 or 5

Reply:
In your case region, city, office can be null. Is ok for you?

Reply:
Made changes but still have an issues with validation